README.md

```diff
@@ -327,6 +327,24 @@ To install the completions for fish, run
 sudo cp alacritty-completions.fish $__fish_datadir/vendor_completions.d/alacritty.fish
 ```
 
+## Terminfo
+
+The terminfo database contains entries describing the terminal
+emulator's capabilities. Programs need these in order to function
+properly.
+
+Alacritty should work with the standard `xterm-256color` definition,
+but to allow programs to make best use of alacritty's capabilities,
+use its own terminfo definition instead.
+
+Note that the default configuration **requires** alacritty's terminfo.
+
+To install alacritty's terminfo entry globally:
+
+```sh
+sudo tic -e alacritty,alacritty-direct alacritty.info
```

Comment:
I'm assuming most package maintainers make the decision to do this for the user or not. My question is if the supported way to install alacritty is through cargo with `cargo install alacritty`, should we have the build script do this for them? Or at the very least, post-install message 'Hey you should run this'?

Reply:
This is a really good question, and I'm not sure about the right approach. It would kind of be too bad if every time the build script ran, a new terminfo definition was installed (at least for Alacritty devs).

While `cargo install alacritty` would certainly be nice for Rust programmers, I think the long-term "supported" way to install it will be through package managers or binaries downloaded from the project. In those cases, we may have a bit more flexibility on packaging terminfo definitions.